src/cmd/flux-mini.py

```diff
@@ -218,6 +230,29 @@ def create_parser(self):
             help="Set shell option (multiple use OK)",
             metavar="KEY=VAL",
         )
+        parser.add_argument(
+            "--setattr",
+            action="append",
+            help="Set job attribute (multiple use OK)",
+            metavar="KEY=VAL",
+        )
+        parser.add_argument(
+            "--output",
+            type=str,
+            help="Redirect job stdout from KVS to FILENAME",
+            metavar="FILENAME",
+        )
+        parser.add_argument(
+            "--error",
+            type=str,
+            help="Redirect job stderr from KVS to FILENAME",
+            metavar="FILENAME",
+        )
+        parser.add_argument(
+            "--label",
```

Comment:
Nit: I kinda prefer `--label-io` only because `--label` seems ambiguous as to what is being labelled.  Is the job getting a label?   Not a hill I will die on though, so please feel free to disregard.

Reply:
I vote --label-io as well

Reply:
I vote --label-io as well

Reply:
On the subject of option names, would it be more descriptive to rename
* `--cpus-per-task` to `--cores-per-task`?
* `--time` to `--time-limit` or `--max-duration`?

Reply:
I think both of those are a good idea. (for `--time`, `--time-limit` is preferable just because the terminology will be more familiar to users)